test/integration/imagechange_buildtrigger_test.go

```diff
@@ -465,6 +465,8 @@ func TestMultipleImageChangeBuildTriggers(t *testing.T) {
 	}
 	defer watch2.Stop()
```

Comment:
I see, poor naming again. I visually compared char-by-char and could swear to death that they were the same... but no, there're not.

We should call them `buildWatcher`, `buildConfigWatcher` in compassion for the next people reading this test case...